solr/solr-ref-guide/modules/configuration-guide/pages/implicit-requesthandlers.adoc

```diff
@@ -74,17 +74,6 @@ This handler must have a collection name in the path to the endpoint.
 |`solr/<collection>/admin/luke` |{solr-javadocs}/core/org/apache/solr/handler/admin/LukeRequestHandler.html[LukeRequestHandler] |`_ADMIN_LUKE`
 |===
 
-MBeans:: Provide info about all registered {solr-javadocs}/core/org/apache/solr/core/SolrInfoBean.html[SolrInfoMBeans].
```

Comment:
removed; we'll ensure this removal is stated elsewhere for Solr 10 elsehwere.  Found via broken link detector